autorel/clients/http_client.py

```diff
@@ -102,7 +81,7 @@ def carryRequest(self,method='GET',url='',headers={},params={},payload={}):
         '''
         method = method.lower()
         self._makeConnection()
```

Comment:
You left this here, please use `git revert` next time.
I know, you fixed this in ef2b2edb, but we usually prefer atomic commits with consistent small changes.